.compose.env.example

```diff
@@ -13,6 +13,9 @@ DEV_SOURCE_PATH='galaxy_ng'
 # and unpin galaxy_ng/setup.py requirements
 LOCK_REQUIREMENTS=1
 
+## Reinstall checkouts on every container launch
+WITH_DEV_INSTALL=1
```

Comment:
This will require that everyone updated their local `.compose.env` adding this variable as it has no default anymore.

CAn you please add it on the lines here https://github.com/ansible/galaxy_ng/blob/master/compose#L44-L51 

```bash
declare -xr WITH_DEV_INSTALL="${WITH_DEV_INSTALL:-1}"
```

So it will be set as default to `1` if it is not present on the environment, that **hack** is done on `./compose` file because docker-compose doesn't allow us to provide  a default value on the yaml file.